README.md

```diff
@@ -141,7 +141,6 @@ A curated list of awesome Open Source Software made in Israel, and by Israelis.
 * [Cloudify](https://github.com/cloudify-cosmo/)
 * [Nexmo](https://github.com/Nexmo)
 * [ZOOZ](https://github.com/Zooz)
-* [Walla! CODE](https://github.com/wallacode)
```

Comment:
That still exists, along with contribution history for all folks involved. I think that's valuable to keep in this list.